apps/docs/pages/guides/platform/exhaust-ram.mdx

```diff
@@ -0,0 +1,53 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'exhaust-ram',
+  title: 'High Memory Usage',
+  description:
+    'Learn what high RAM usage could mean for your Supabase instance and what could have caused it.',
+}
+
+Learn what high RAM usage could mean for your Supabase instance and what could have caused it.
+
+## Base Memory Usage
+
+You might observe elevated memory usage, even with little to no load. Besides Postgres, a wide range of services is running under the hood resulting in an elevated base memory usage. Especially on the smallest compute instance that comes with 1 GB of RAM, it is not unusual for your project to have a base memory usage of ~50%.
+
+## Issues with High Memory Usage
+
+Every Supabase project runs in its dedicated virtual machine. Your instance will have a different set of hardware provisioned depending on your [compute add-on](https://supabase.com/docs/guides/platform/compute-add-ons). Your hardware may not be suitable for the intended workload and may experience high RAM usage.
+
+High RAM usage could come with a range of issues:
+
+- degraded performance overall as your instance might have to rely more on swap
+- the operating system might start killing processes as your system runs out of memory
+- in rare cases, your instance may become unresponsive
+
+## Monitor your RAM
+
+To check your RAM usage on the Supabase Platform, head over to [database health in the reports section](https://app.supabase.com/project/_/reports/database).
+
+<Admonition type="note">
+
+High memory usage doesn't necessarily mean that your instance is at risk. Memory that is used for caching and buffers improves data access speed. But if you notice less performance alongside high memory usage, your memory usage might be unhealthy.
+
+</Admonition>
```

Comment:
Wondering if it would be worth moving this higher up - possibly under the base memory usage (so that customers know high base usage can be okay)?

Reply:
Yes, good point. This is probably something they should know about first. Will move it even above the base memory usage 👍

<img width="885" alt="image" src="https://github.com/supabase/supabase/assets/1732217/5efbf668-f047-40a6-bb18-8be4c935174a">